tsconfig.json

```diff
@@ -46,12 +46,10 @@
     "src/data/api-service.js",
     "src/data/usage-data-service.js",
     "src/store/user.js",
-    "src/utils/decode-data.js",
```

Comment:
To get `nuxt.config.ts` included in the type checking we'll need to change the first entry in `includes` to just `**/*.ts` I think. Right now it's only type checking stuff inside the `src` file (which isn't good considering we'll be adding TypeScript to the `test` directory as well with the visual regression stuff.